proxy.py

```diff
@@ -1521,8 +1519,10 @@ def load_plugins(plugins: bytes) -> Dict[bytes, List[type]]:
         plugin = plugin.strip()
         if plugin == b'':
             continue
+        # coverage test
         module_name, klass_name = plugin.rsplit(DOT, 1)
-        module = importlib.import_module(text_(module_name))
+        module_name = __name__ if module_name == b'proxy' else text_(module_name)
+        module = importlib.import_module(__name__)
```

Comment:
I see tests are failing, perhaps you want to use `sys.modules[__name__]` when `module_name == b'proxy'`.  Wdyt?

Reply:
The test is successful but coverage is decreased

Reply:
Aah, I didn't realize.  Coverage is ok, I did mention exception for it here https://github.com/abhinavsingh/proxy.py#pull-request, probably there is a .coveragerc config to only fail build if coverage decreases by, say more than `>1%`.

Reply:
I didn't notice it. 


>probably there is a .coveragerc config to only fail build if coverage decreases by

That would be great it there is.

Is there anything else to suggest?

Reply:
Thanks for the PR.  All looks good 👍